solutions/pytorch_tests/README.md

```diff
@@ -0,0 +1,26 @@
+### 1. test_torch.py - Basic tests for PyTorch functionality.
+- 3 failed cases
```

Comment:
Provide a little introduction to the PyTorch test suite, including to the passed ones,  before talking about the failed test cases.

Reply:
Fixed.